vscode-extension/package.json

```diff
@@ -59,7 +59,7 @@
       },
       {
         "command": "vscode-aiconfig.setupEnvironmentVariables",
```

Comment:
nit: Can we also update this to be `vscode-aiconfig.setApiKeys`

Reply:
Will do in follow-up